tests/document_ops.rs

```diff
@@ -211,32 +212,209 @@ fn doc_create_with_policy_grants() -> Result<(), IronOxideErr> {
 }
 
 #[test]
-fn doc_create_with_explicit_grants() {
-    let sdk = init_sdk();
+fn doc_edek_encrypt_with_policy_grants() -> Result<(), IronOxideErr> {
+    // policy assumed for this test
+    /*
```

Comment:
I don't feel like we need another test on policy. I know it's tempting to just duplicate the other tests for this, but it causes any changes to make changing the tests onerous. Is there a way we can simplify this test?

Reply:
After review, I agree. Deleted!